Makefile

```diff
@@ -4,15 +4,15 @@
 SHELL=/bin/bash
 
 ### 1. change these settings or override with env variables
-CONFIG=configs/config.prod.yml
-CONDA_PATH=../mambaforge
-SNAKEMAKE_OUTPUT_CACHE=../cache
-PROFILE=local
+CONFIG?=configs/config.prod.yml
```

Comment:
My only concern with those is that there might be some random envs on Berlin cluster that will override the settings. I guess let's see. Let's merge this and check afterwards.

Reply:
@gregtatum or you can check before, since you already have access to the cluster

Reply:
If that happens I'll follow up by prefixing them.